Bottom: a555f0a7acdf23b66274cc4ef7645f552a98d0f5
Top:    4bbcb14e0681aad0a350fedd37f6fa80dd36ffe4
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-06-05 18:44:15 +0100

find.c: The User varaible needs to die...

Reduce its usage to where absolutely necessary... oh, what a surprise,
another variable passing state down into add_object_to_list ()


---

diff --git a/src/find.c b/src/find.c
index ac8e01d..f11f3d6 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3090,7 +3090,7 @@ LookupConnection (Coord X, Coord Y, bool AndDraw, Coord Range, int flag,
    */
   ListStart (type, ptr1, ptr2, ptr3, flag);
   DoIt (flag, AndRats, AndDraw);
-  if (User)
+  if (AndDraw)
     IncrementUndoSerialNumber ();
   User = false;
